content/en/templates/taxonomy-templates.md

```diff
@@ -292,7 +307,9 @@ The following example displays all terms in a site's tags taxonomy:
 ```
 <ul id="all-tags">
   {{ range $name, $taxonomy := .Site.Taxonomies.tags }}
-    <li><a href="{{ "/tags/" | relLangURL }}{{ $name | urlize }}">{{ $name }}</a></li>
+  {{ with $.Site.GetPage "taxonomy" "tags" $name }}
```

Comment:
(same as above)

Though, I don't quite like hard-coding "tags" taxonomy in templates. I might write a PR suggesting to change this example to a more generic version after this gets merged.